### PR TITLE
[2.13] Allow two warnings

### PR DESCRIPTION
**Pull request by fedinskiy**
Fixes failures due to two upstreams issues:
Missing classes: https://github.com/quarkusio/quarkus/issues/30508
Missing builder image: https://github.com/quarkusio/quarkus/issues/30738

### CODE REVIEW

**Comment by rsvoboda** [comment]
Missing builder image seems to be fixed now

File: testsuite/src/it/java/io/quarkus/ts/startstop/utils/Commons.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package io.quarkus.ts.startstop.utils;
+
+import java.util.regex.Pattern;
+
+public class Commons {
+    static Pattern NETTY_HANDLERS=Pattern.compile(".*Could not register io.netty.handler.codec.*"); //https://github.com/quarkusio/quarkus/issues/30508
+    static Pattern NO_IMAGE = Pattern.compile(".*Tag 22.3-java17 was deleted or has expired.*"); //https://github.com/quarkusio/quarkus/issues/30738
```
Comment (rsvoboda) [inline testsuite/src/it/java/io/quarkus/ts/startstop/utils/Commons.java:7]: IMHO not needed anymore

File: testsuite/src/it/java/io/quarkus/ts/startstop/utils/Commons.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package io.quarkus.ts.startstop.utils;
+
+import java.util.regex.Pattern;
+
+public class Commons {
+    static Pattern NETTY_HANDLERS=Pattern.compile(".*Could not register io.netty.handler.codec.*"); //https://github.com/quarkusio/quarkus/issues/30508
```
Comment (rsvoboda) [inline testsuite/src/it/java/io/quarkus/ts/startstop/utils/Commons.java:6]: Better to put it into `WhitelistLogLines` and drop `Commons` class

**Review by rsvoboda — CHANGES_REQUESTED** [review]

**Review by pjgg — APPROVED** [review]
LGTM

**Pull request merged**